kedro-datasets/.readthedocs.yaml

```
@@ -12,8 +12,7 @@ build:
   jobs:
     pre_install:
       - python -m pip install "uv==0.1.13"
-      - export UV_HTTP_TIMEOUT=600
-      - uv pip install --system --upgrade sphinx readthedocs-sphinx-ext "kedro-datasets[docs,test] @ ./kedro-datasets"
+      - export UV_HTTP_TIMEOUT=1000 && uv pip install --system --upgrade sphinx readthedocs-sphinx-ext "kedro-datasets[docs,test] @ ./kedro-datasets"
```

```suggestion
      - UV_HTTP_TIMEOUT=1000 uv pip install --system --upgrade sphinx readthedocs-sphinx-ext "kedro-datasets[docs,test] @ ./kedro-datasets"
```

I think this should work